ci-operator/step-registry/rhba/interop-tests/rhba-interop-tests-commands.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export TEST_COLLECT_BASE_DIR=${ARTIFACT_DIR}
```

Comment:
Is this needed? If so could you also add it to env variables in readme

Reply:
Added